src/pages/payment-methods/_mobile-expanded-list.js

```diff
@@ -116,7 +110,9 @@ const MobileExpandedList = ({ is_crypto, is_fiat_onramp, is_dp2p, locale, paymen
                     {/* first row */}
                     <StyledRow jc="space-between" ai="center">
                         <StyledItemDiv>
-                            <Header type="subtitle-2">{localize('Currencies')}</Header>
+                            <Header as="p" type="subtitle-2">
```

Comment:
Added `as "p"` tag per comment in pervious PR which overlooked 🙏 

https://github.com/binary-com/deriv-com/pull/2495/files#r785597175

Reply:
Why don't we use <text> instead? what's the reason to add `p` ?

Reply:
> Why don't we use `<text>` instead? what's the reason to add `p` ?

Hi @sara-fs many thanks for reviewing , as I checked this with Sean seems `<text>` is now deprecated as `Header` component has the correct size mapping from the design team. 🙏